corehq/apps/accounting/templates/accounting/30_days.html

```diff
@@ -0,0 +1,42 @@
+{% load i18n %}
+<p>
+    {% blocktrans %}
+        Dear {{ invoice.get_domain }} administrator,
+    {% endblocktrans %}
+</p>
+
+<p>
+    {% blocktrans %}
+        Your billing account has unpaid CommCare Billing Statements which are more than 30 days overdue. The balance currently owed is: ${{ total }}.
+    {% endblocktrans %}
+</p>
+
+<p>
+    {% blocktrans %}To see the full list of unpaid invoices you can follow this link: {% endblocktrans %}<a href="{{ statements_url }}">{{  statements_url }}</a>
+<ul>
+    <li>{% blocktrans %}As a reminder, payments can be made via credit card, Electronic Fund Transfer (EFT), or check by following these <a href="https://confluence.dimagi.com/display/commcarepublic/How+to+Pay+by+Credit+Card+or+Wire+Payment">instructions</a>.{% endblocktrans %}</li>
+    <li>{% blocktrans %}You can also pre-pay for several months at any time by following <a href="https://confluence.dimagi.com/display/commcarepublic/CommCare+Pricing+FAQs#CommCarePricingFAQs-Pre-payment">these steps</a>.{% endblocktrans %}</li>
+</ul>
+</p>
+
+<p>
+    {% blocktrans %}
+        <b><i>Please make a payment as soon as possible for the amount due.</i></b>
+        If you are not able to make a payment, your account will be downgraded to our free Community edition on {{ date_60 }}.
+        As of this date, you will lose access to direct email support from CommCare support team.
+    {% endblocktrans %}
+</p>
+
+<p>
+    {% blocktrans %}
+        If you have any questions or if you think your recent payments haven’t been reflected in this balance, please don't hesitate to contact billing-support@dimagi.com.
```

Comment:
also that email should be defined in a setting.  @calellowitz let me know if you need help finding the appropriate setting